test/calendar_test.js

```diff
@@ -153,10 +153,10 @@ describe("Calendar", () => {
       state = {
         weeks: Calendar.monthWeekTable(2017, 1),
         holidays: [{
-          start: "2017-01-02T00:00:00.000Z",
+          start: "2016-12-26T00:00:00.000Z",
```

Comment:
2 reasons:
1. this is the actual bug we have
2. this was actually the same test than below